### PR TITLE
add token label support

### PR DESCRIPTION
**Pull request by C0MM4ND**

### CODE REVIEW

**Comment by brianleect** [comment]
@C0MM4ND Thanks for the PR it looks good. Do you think you would be able to run the scraper and populate the token section as well?

**Review by brianleect — COMMENTED** [review]
^ Populate tokens

**Comment by C0MM4ND** [comment]
Thanks for your review.

I've tested some of the account and token labels, but not all.
Crawling all labels requires my laptop continuously open for a relative long time, but I'm on a outdoor business trip during these days.
So let me convert this PR back to draft. I'll re-open after the new commit.

File: main.py [hunk 1]
```diff
@@ -85,31 +95,35 @@ def getLabel(label, type='single'):
...
     # iterating over all files
-    for files in os.listdir('./data'):
+    for files in os.listdir('./accounts'):
         if files.endswith('json'):
             print(files)  # printing file name of desired extension
-            with open('./data/{}'.format(files)) as f:
+            with open('./accounts/{}'.format(files)) as f:
                 dictData = json.load(f)
                 for address, nameTag in dictData.items():
                     if address not in combinedJSON:
```
Comment (brianleect) [inline main.py:129]: Any thoughts regarding the need to combine for tokens as well or just accounts being sufficient?

**Review by brianleect — CHANGES_REQUESTED** [review]
Thanks for the PR, everything looks good. Just need to address whether it makes sense to add token data for function combineAllJson()

**Review by brianleect — APPROVED** [review]
Looks good!

**Pull request merged**